process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+# Becoming an OpenSSF Project
+
+The OpenSSF's mission is to inspire and enable the community to secure the open source software we all depend on. OpenSSF's top priority is the security of existing open source projects however, there are missing pieces in the tooling needed to improve open source project security. These pieces are the projects OpenSSF aims to steward and support.
+
+## Project Life Cycle
+
+New projects to the OpenSSF, and progression through the project lifecycle, are approved by the Technical Advisory Committee (TAC). A project's oversight is provided by either the Technical Advisory Committee (TAC) or a specific Working Group (WG). When a Project reports into a specific WG, that WG can support the Project's progression and provide recommendations to the TAC.
+
+Projects follow the Sandbox, Incubating, Graduated, and Archived lifecycle stages defined below. Projects that seek widespread adoption and end user use are expected to progress through the stages. Projects coming to OpenSSF that already meet the entry requirements may enter the Incubating stage directly.
+
+<!-- TOC -->
+
+  - [Stages - Definitions & Expectations](#stages---definitions--expectations)
+      - [Sandbox](#sandbox)
+      - [Incubating](#incubating)
+      - [Graduated](#graduated)
+      - [Archived](#archived)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+A project's lifecycle stage is a signal of maturity level. Projects progress through the lifecycle by demonstrating their maturity in technical vision, impact to the open source community (such as adoption), and sustainability in governance and contributors. All projects, at all stages, are expected to abide by the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/).
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage experimentation and open collaboration that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if this is desired (there may be cases where pursuit of different approaches is desirable)
+* Nurture Projects
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Project Responsibilities
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project)
+* Provides project updates to OpenSSF Marketing Committee as requested
+
```

Comment:
```suggestion
* Follow security best practices (as recommended by the OpenSSF and others)
* Maintain a point of contact for vulnerability reports

```
My 2c: I think we need to set the bar high here for *any* affiliated projects. We should be leading by example rather than deferring this work to later.

Reply:
agreed with Bob.  Any OSSF projects should follow our own guidance (Badges, SLSA, sigstore, etc).

Reply:
@bobcallaway I'm not sure whether you mean that this should apply to enter Sandbox too. I would say not because that's essentially the point of Sandboxing: ramping up to the level expected of a Project in Incubation and beyond.

Reply:
+1 to this.

Reply:
@lehors I did intend for this to apply to Sandbox projects - because we need to lead by example that best practices should be adopted ASAP (rather than being technical debt on the backlog). if we find that this bar is too high, then I would propose we focus on closing that gap via additional project/WG activity, not by lowering the bar. While the OSSF mission is largely focused on existing projects, IMO we should also be working to ensure better defaults are adopted sooner with near-zero friction.

Reply:
The way this is written it does not preclude the OpenSSF from defining a lower best practices bar for Sandbox projects.  That can be determined outside of this and should (IMHO) be the discretion of the TAC to manage as a Sandbox consideration and criteria when choosing to vote a project into that status.

Within CNCF, we're iterating on what we look for from sandbox projects, as better security defaults and templates become available to the community, our minimum expectations are being raised as it is no longer a "high bar" or does not introduce "friction" to do specific, security-oriented practices.